etna/records/models.py

```diff
@@ -224,15 +224,28 @@ def catalogue_source(self) -> str:
 
     @cached_property
     def description(self) -> str:
-        if raw := self._get_raw_description():
+        """
+        Returns the records full description value with all HTML left intact.
+        """
+        return mark_safe(self._get_raw_description(use_highlights=False))
+
+    @cached_property
+    def listing_description(self) -> str:
+        """
+        Returns a version of the record's description that is safe to use
+        anywhere. When highlight data is provided by the API, <mark> tags
+        will be left in-tact, but and other HTML is stripped.
+        """
+        if raw := self._get_raw_description(use_highlights=False):
```

Comment:
```suggestion
    @cached_property
    def highlight_description(self) -> str:
        """
        Returns a version of the record's description that is safe to use
        anywhere. When highlight data is provided by the API, <mark> tags
        will be left in-tact, but and other HTML is stripped.
        """
        if raw := self._get_raw_description(use_highlights=True):
```
Field name with `highlight_` in it may serve the purpose of using description with highlight. Thoughts?
The `use_highlights` param requires to be enabled to True in order to use it in `_get_raw_description`

Reply:
The purpose of `listing_description` is a bit more nuanced than just the use of highlights (the stripping of non-highlight HTML, for example). It can also be used in listings where highlights aren't even present, so `highlight_description` in that context would be misleading.